packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -591,6 +594,24 @@ abstract class FlutterCommand extends Command<void> {
       valueHelp: 'foo=bar',
       splitCommas: false,
     );
+    /// define support config json file; the value is add to `dart-define` constants
```

Comment:
nit: this should be a comment not a dart doc 
`// define support config json file; the value is add to `dart-define` constants`

Reply:
Get it ,i fix it